charts/umbrella/README.md

```diff
@@ -330,7 +338,28 @@ TBD.
 
 #### Get to know the Portal
 
-Perform first login and send out an invitation to a company to join the network (SMTP account required to be configured in custom values.yaml file).
+Perform first login and send out an invitation to a company to join the network (SMTP account required to be configured in custom values.yaml file). Since the onboarding process requires the clearinghouse to work properly, but clearinghouse currently isn't available as a FOSS application you can skip the step with the following SQL Script which must be executed against the portal database.
+
+```sql
+WITH applications AS (
+    SELECT distinct ca.id as Id, ca.checklist_process_id as ChecklistId
+    FROM portal.company_applications as ca
+             JOIN portal.application_checklist as ac ON ca.id = ac.application_id
+    WHERE 
+      ca.application_status_id = 7 
+    AND ac.application_checklist_entry_type_id = 6
+    AND (ac.application_checklist_entry_status_id = 4 OR ac.application_checklist_entry_status_id = 1)
+),
+updated AS (
+ UPDATE portal.application_checklist
+     SET application_checklist_entry_status_id = 3
+     WHERE application_id IN (SELECT Id FROM applications)
+     RETURNING *
+)
+INSERT INTO process_steps (id, process_step_type_id, process_step_status_id, date_created, date_last_changed, process_id, message)
```

Comment:
The schema needs to be set, or else it won't find the table. (portal.process_steps)